client/hostinfo_unix.cpp

```diff
@@ -1584,6 +1584,45 @@ int HOST_INFO::get_memory_info() {
     return 0;
 }
 
+#if LINUX_LIKE_SYSTEM
+// gather libc version from the system installed ldd binary
+// a usual first line looks like: ldd (Debian GLIBC 2.24-11+deb9u1) 2.24
+// where the last part is copied to version and the string between parenthesis
+// is copied to extra_info
+//
+// return 0 if at least version could be found (extra_info may remain empty)
+// return 1 if ldd couldn't be opened or no version infromation was found
+//
+int get_libc_version(string& version, string& extra_info) {
```

Comment:
I'm no big fan of BOINC's error codes but those would be nicer than magic number. BOINC_SUCCESS and ERR_NOT_FOUND or something.

Reply:
You're right. I'm going to change it to use BOINC_SUCCESS and ERR_NOT_FOUND.